types/mytime.go

```diff
@@ -117,6 +117,31 @@ func (t MysqlTime) GoTime(loc *gotime.Location) (gotime.Time, error) {
 	return tm, nil
 }
 
+// ConvertTimeZone converts the mysql time value from one timezone to another.
+// `t` must match the timezone of `from`
+func (t MysqlTime) convertTimeZone(from, to *gotime.Location) (MysqlTime, error) {
```

Comment:
Why we need this function? There is already a `ConvertTimeZone` in time.go

Reply:
And MysqlTime has no concept of `TimeZone`, just like MySQL

Reply:
done~deleted.